webapp/src/components/AssetPage/OwnersTable/OwnersTable.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useEffect, useState } from 'react'
+import { Table, Loader, Icon, Row, Pagination } from 'decentraland-ui'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { Link } from 'react-router-dom'
+import { locations } from '../../../modules/routing/locations'
+import { nftAPI } from '../../../modules/vendor/decentraland'
+import { LinkedProfile } from '../../LinkedProfile'
+import ListedBadge from '../../ListedBadge'
+import {
+  OrderDirection,
+  OwnersFilters,
+  OwnersResponse,
+  OwnersSortBy,
+  Props
+} from './OwnersTable.types'
+import styles from './OwnersTable.module.css'
+
+const ROWS_PER_PAGE = 6
+
+const OwnersTable = (props: Props) => {
+  const { asset, orderDirection = OrderDirection.ASC } = props
+
+  const [owners, setOwners] = useState<OwnersResponse[]>([])
+  const [total, setTotal] = useState(0)
+  const [page, setPage] = useState(1)
+  const [totalPages, setTotalPages] = useState(1)
+  const [isLoading, setIsLoading] = useState(false)
+
+  // We're doing this outside of redux to avoid having to store all orders when we only care about the first ROWS_PER_PAGE
+  useEffect(() => {
+    if (asset) {
+      setIsLoading(true)
+      let params: OwnersFilters = {
+        contractAddress: asset.contractAddress!,
+        itemId: asset.itemId!,
+        first: ROWS_PER_PAGE,
+        skip: (page - 1) * ROWS_PER_PAGE,
+        sortBy: OwnersSortBy.ISSUED_ID,
+        orderDirection: orderDirection
+      }
+      nftAPI
+        .getOwners(params)
+        .then(response => {
+          setTotal(response.total)
+          setOwners(response.data)
+          setTotalPages(Math.ceil(response.total / ROWS_PER_PAGE) | 0)
+        })
+        .finally(() => setIsLoading(false))
+        .catch(error => {
+          console.error(error)
+        })
+    }
+  }, [asset, setIsLoading, setOwners, page, orderDirection])
+
+  return (
+    <div className={styles.OwnersTable}>
+      {isLoading ? (
+        <Loader />
```

Comment:
```suggestion
        <Loader active />
```
`active` prop is missing, otherwise it won't be shown